Marlin/ubl_G29.cpp

```diff
@@ -1341,15 +1339,7 @@
           // Also for round beds, there are grid points outside the bed that nozzle can't reach.
           // Prune them from the list and ignore them till the next Phase (manual nozzle probing).
 
-//        if ((probe_as_reference && position_is_reachable_by_probe_raw_xy(mx, my)) || position_is_reachable_raw_xy(mx, my))
-//          continue;
-//
-//        THE ABOVE CODE IS NOT A REPLACEMENT FOR THE CODE BELOW!!!!!!!
-//
-          bool reachable = probe_as_reference ?
-                             position_is_reachable_by_probe_raw_xy( mx, my ) :
-                             position_is_reachable_raw_xy( mx, my );
-          if ( ! reachable )
+          if ( ! (probe_as_reference ? position_is_reachable_by_probe_raw_xy(mx, my) : position_is_reachable_raw_xy(mx, my)) )
```

Comment:
I did figure out how to automatically do 2 space indentation...  And to replace tabs with spaces...
It might just stop putting trailing white space on a line now...  We will see.   But that is easy to search for...   " $" will find a space and "\t$" will find a tab.  But in fact, we don't even use tabs so searching for any "\t" means something has to be deleted.